conda/resolve.py

```diff
@@ -923,19 +936,18 @@ def mysat(specs, add_if=False):
         log.debug('Additional package channel/version/build metrics: %d/%d/%d',
                   obj5a, obj5, obj6)
 
-        # Maximize timestamps
-        eq_t.update(eq_req_t)
-        solution, obj6t = C.minimize(eq_t, solution)
-        log.debug('Timestamp metric: %d', obj6t)
-
         # Prune unnecessary packages
         eq_c = r2.generate_package_count(C, specm)
         solution, obj7 = C.minimize(eq_c, solution, trymax=True)
         log.debug('Weak dependency count: %d', obj7)
 
-        def clean(sol):
-            return [q for q in (C.from_index(s) for s in sol)
-                    if q and q[0] != '!' and '@' not in q]
+        multiple_solutions = has_multiple_simple_solutions(solution)
+        if multiple_solutions:
```

Comment:
Is there an easy way to show how this speeds up a solve? Just curious

Reply:
If `multiple_solutions` is False (i.e. there is a single solution) then the block of code is not executed.  That block, particularly the line `solution, obj6t = C.minimize(eq_t, solution)` line can take considerable time if a large number of packages are being considered.